extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -535,6 +537,67 @@ def from_dict(self, model):
         self._c_lw = copy.deepcopy(model['_c_lw'])
         self._c_l = copy.deepcopy(model['_c_l'])
 
+    def _make_json_serializable(self, model):
```

Comment:
There is too much duplication within this function; please find a way to do this using a for loop and fewer chunks of copy/pasted code.